src/motor_hardware.cc

```diff
@@ -79,8 +79,18 @@ MotorHardware::MotorHardware(ros::NodeHandle nh, CommsParams serial_params,
     prev_pid_params.pid_derivative = -1;
     prev_pid_params.pid_denominator = -1;
     prev_pid_params.pid_moving_buffer_size = -1;
+    prev_pid_params.max_speed_fwd = -1;
+    prev_pid_params.max_speed_rev = -1;
```

Comment:
Change the name of this member, pid_params and prev_pid params are incorrected as there are a lot more than just PID values

Reply:
I'll make it prev_firmware_params as that structure holds all firmware params as the class implies using FirmwareParams firmware_params